code/ui/components/src/new/Input/Input.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { forwardRef } from 'react';
+import { styled } from '@storybook/theming';
+
+interface InputProps {
```

Comment:
Consider adding PropTypes for better type checking and to make the code more self-documenting. [medium]

Reply:
euh ... no.